apps/system/test/unit/nfc_handover_manager_test.js

```diff
@@ -302,6 +302,23 @@ suite('Nfc Handover Manager Functions', function() {
       assert.equal(1, nfcHandoverManager.sendFileQueue.length);
     });
 
+    test('Sending aborts when another file is transmitted concurrently',
+      function() {
+
+      this.sinon.stub(Service, 'query', function(queryString) {
```

Comment:
Use MockService and implement MockService.mIsFileTransferInProgress instead of stubbing here